tests/api/exclude/pyiceberg-0.6.0.yaml

```diff
@@ -0,0 +1,47 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# The format of this file is documented at
+# https://cwiki.apache.org/confluence/display/INFRA/Git+-+.asf.yaml+features
+
+exclude:
+- obj_path: pyiceberg.avro.decoder_fast.CythonBinaryDecoder
+  kind: CLASS_REMOVED_BASE
+- obj_path: pyiceberg.table.create_mapping_from_schema
```

Comment:
This one seems correct, but not exactly what I expected: https://github.com/apache/iceberg-python/commit/fd9dc8837d4b9054de2e51ee360ead517f471271#diff-23e8153e0fd497a9212215bd2067068f3b56fa071770c7ef326db3d3d03cee9bL89

Reply:
Yeah, that's interesting that it treats imports from other modules to be public objects as well